src/core/utils/helpers/general.ts

```diff
@@ -372,7 +372,7 @@ export const getAllFaustIds = (manifestations: Manifestation[]) => {
 export const getScrollClass = (modalIds: string[]) => {
   return modalIds.length > 0 ? "scroll-lock-background" : "";
 };
-export const dataIsNotEmpty = (data: unknown[]) => Boolean(data.length);
+export const isDataEmpty = (data: unknown[]) => Boolean(!data.length);
```

Comment:
:+1:

We could also consider replacing this with [`_.isEmpty()`](https://lodash.com/docs/4.17.15#isEmpty).